packer/conf/buildkite-agent/hooks/pre-command

```diff
@@ -5,7 +5,7 @@ set -eu -o pipefail
 # AWS ECR login support
 
 # For logging into the current AWS account’s registry
-if [[ -n "${AWS_ECR_LOGIN:-}" ]] ; then
+if [[ "${AWS_ECR_LOGIN:-true}" =~ ^(true|1)$ ]] ; then
```

Comment:
I think for a little while we supported `AWS_ECR_LOGIN=<region>`, which would break this. Something to keep in mind for the release notes!